dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/validation/EncodingValidator.java

```diff
@@ -34,6 +38,26 @@ private EncodingValidator() {}
     /** {@inheritDoc} */
     @Override
     public void validate(ValidationHelper helper, Context context, String key, Encoding t) {
-        // TODO Auto-generated method stub
+
+        if (t != null) {
+
+            if (t.getStyle() != null) {
+
+                Style style = t.getStyle();
+                Boolean explode = t.getExplode();
+                String[] styleExplode = { "spaceDelimited", "pipeDelimited", "deepObject" };
+
+                if (style.toString().equals("form") && !explode) {
```

Comment:
Not sure if this check is required. Where does the spec mention it?

I only found this in the spec: "When style is form, the default value is true. For all other styles, the default value is false. "
But it only talks about 'default' value. The user can still set the value to something else.

Reply:
My initial goal was to address issue #7343 from CL, but you are right, that would validated user specified values, and not the default as we want. I will remove this check.

Reply:
...which actually leaves nothing to validate in Encoding :)